apps/src/templates/progress/ProgressBubble.story.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ProgressBubble from './ProgressBubble';
+
+export default storybook => {
+  storybook
+    .storiesOf('ProgressBubble', module)
+    .addStoryTable([
```

Comment:
It'd be pretty easy here to generate a storytable that covers _all_ bubble colors - might be nice to have.